src/DataTypes/Base.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace NerdsAndCompany\Schematic\DataTypes;
+
+use NerdsAndCompany\Schematic\Interfaces\DataTypeInterface;
+
+abstract class Base implements DataTypeInterface
+{
+    /**
+     * {@inheritdoc}
+     */
+    abstract public function getMapperHandle(): string;
+
+    /**
+     * {@inheritdoc}
+     */
+    abstract public function getRecords(): array;
+
+    /**
+     * {@inheritdoc}
+     */
+    public function afterImport()
```

Comment:
Maybe I should explicitly call it clearCache or something like it as import is responsibility of the mapper